src/Toolkit/Toolkit/UI/Controls/BasemapGallery/BasemapGalleryController.cs

```diff
@@ -17,35 +17,124 @@
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Collections.Specialized;
+using System.ComponentModel;
 using System.Linq;
 using System.Threading.Tasks;
 using Esri.ArcGISRuntime.Mapping;
 using Esri.ArcGISRuntime.Portal;
-using Esri.ArcGISRuntime.Toolkit.UI.Controls;
+
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Esri.ArcGISRuntime.Toolkit.Xamarin.Forms")]
```

Comment:
Shouldn't this be removed?

Reply:
I did try removing that, but that would require making more API public on `BasemapGalleryItem` than we're comfortable with.